lib/ansible/modules/cloud/azure/azure_rm_mysqlserver.py

```diff
@@ -0,0 +1,368 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_mysqlserver
+version_added: "2.5"
+short_description: Manage MySQL Server instance
+description:
+    - Create, update and delete instance of MySQL Server
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    name:
+        description:
+            - The name of the server.
+        required: True
+    sku:
+        description:
+            - The SKU (pricing tier) of the server.
+        suboptions:
+            name:
+                description:
+                    - The name of the sku, typically, a letter + Number code, e.g. P3.
+            tier:
+                description:
+                    - "The tier of the particular SKU, e.g. Basic. Possible values include: 'Basic', 'Standard'"
```

Comment:
is `Basic` and `Standard` the only values, if so change it to list these options and set a default. Also try and make values in a snake case fashion so all lower case and `_` instead of spaced.

Also full stop at the end.